README.md

```diff
@@ -99,27 +125,40 @@ module.exports = function (config) {
 -    config.addPlugin(fluidPlugin);
 +    config.addPlugin(fluidPlugin, {
 +        css: {
++              enabled: false,
++        },
++        sass: {
 +              browserslist: '> 1.5%',
++              enabled: true,
 +        }
 +    });
 };
 ```
 
 Default values are as follows:
 
-```json
-{
-    "basePath": "./src/assets/styles",
-    "minify": true,
-    "sourceMap": false,
-    "drafts": {
-        "nesting": true
+```js
+let options = {
+    /* Where should Eleventy look for CSS files to process? */
+    basePath: "./src/assets/styles",
+    /* Should CSS files be processed? */
+    enabled: true,
```

Comment:
From the index.js file it looks like this is disabled by default.

Reply:
Correct— it's not enabled by default.

Reply:
It looks like this documents the default configuration for Sass compilation and that line 145 should be:

```suggestion
    enabled: false,
```